datacats/cli/install.py

```diff
@@ -40,7 +40,8 @@ def install(environment, opts):
             'PORT': None,
             '--background': False,
             '--address': opts['--address'],
-            '--syslog': False})
+            '--syslog': False,
+            '--no-reload': False})
```

Comment:
nit picking here, but.. try to keep these alphabetical or chaos will descend on our beautiful little village and devour everyone.